vault-quota/Design.md

```diff
@@ -13,6 +13,22 @@ The design below only takes into account incremental propagation of space used
 by stored files. It is not complete/verified until we also come up with a validation
 algorithm that can detect and fix discrepancies in a live `vault`.
 
+## operations that effect node size
```

Comment:
Is this section still required? And the other size related changes?

Reply:
re-working the doc based on our change of plans